eng/tools/ToolDescriptionEvaluator/prompts.json

```diff
@@ -1,23 +1,23 @@
 {
   "foundry_agents_connect": [
-    "Query an agent in my AI foundry project"
+    "Query an agent in my Azure AI foundry resource"
```

Comment:
I noticed some bugs in the ToolDescriptionEvaluator related scripts. Let me fix them and update changes unrelated to your stuff in main.

Reply:
If there is a reason why you need to change the Foundry tools' test prompt please let us know. Otherwise, let's keep them as is. I merged a fix to the scripts for generating these testing json data. Please re-run the scripts to update them for your tools.